src/waveform/widgets/waveformwidgettype.h

```diff
@@ -6,21 +6,20 @@ class WaveformWidgetType {
         // The order must not be changed because the waveforms are referenced
         // from the sorted preferences by a number.
         EmptyWaveform = 0,
-        SoftwareSimpleWaveform, //TODO
-        SoftwareWaveform,
-        QtSimpleWaveform,
-        QtWaveform,
-        GLSimpleWaveform,
-        GLFilteredWaveform,
-        GLSLFilteredWaveform,
-        HSVWaveform,
-        GLVSyncTest,
-        RGBWaveform,
-        GLRGBWaveform,
-        GLSLRGBWaveform,
-        QtVSyncTest,
-        QtHSVWaveform,
-        QtRGBWaveform,
+        SoftwareWaveform,        // Filtered
```

Comment:
The removal of `SoftwareSimpleWaveform` offsets all subsequent numbers by -1. But WaveformType is store as this number in mixxx.cfg.

Reply:
Sorry, I overlooked that. Reverted.